Condor-1.1:  Add <alt> for "royalty- free" and use wayback URL

wking opened the pull request:
The extra space is part of the upstream HTML:

```
$ curl -s http://web.archive.org/web/20121029030238/http://research.cs.wisc.edu/condor/license.html | grep -A1 'royalty-$'
   the Software, you are granted a non-exclusive, worldwide, royalty-
   free perpetual license under such patent claims, with the rights
```

I've used an `<alt>` entry to allow both the appropriately space-less version (which I've set as canonical) and the upstream typo'ed version.

The new URL is because the old link redirects to the HTCondor license page:

```
$ curl -sI http://research.cs.wisc.edu/condor/license.html
HTTP/1.1 301 Moved Permanently
Date: Thu, 03 Aug 2017 17:54:54 GMT
Server: Apache
Location: http://research.cs.wisc.edu/htcondor/license.html
Content-Type: text/html; charset=iso-8859-1
```

which does not contain the old Condor 1.1 license.  Walking the archives for the non-HT page, the Condor 1.1 license is [still listed in 2012-10-29][1] but the [next archived result for that page on 2016-05-14 is the 301 redirect][2].

[1]: http://web.archive.org/web/20121029030238/http://research.cs.wisc.edu/condor/license.html
[2]: http://web.archive.org/web/20160514123949/http://research.cs.wisc.edu/condor/license.html

bradleeedmondson commented:
per today's legal call: Brad to accept d624ca982fab8dc3cc8fe791b4037982ccee537f, but instead of 6a6384bd99d178f69e6b035f7ba676a1ad9e6772 we'll add the wayback URL instead of replacing

wking commented:
On Thu, Sep 14, 2017 at 05:31:38PM +0000, Bradlee H. Edmndson wrote:
> … we'll add the wayback URL instead of replacing

Done with d624ca9 → c1b6525.

bradleeedmondson commented:
Oh thanks for that update @wking. I'm a bit of a git novice; how did you swap out commits in the PR without it showing up as a new commit from today (perhaps squashed, perhaps not)?

wking commented:
On Thu, Sep 14, 2017 at 06:02:55PM +0000, Bradlee H. Edmndson wrote:
> … how did you swap out commits in the PR without it showing up as a
> new commit from today (perhaps squashed, perhaps not)?

From the GitHub perspective, you just push to the branch with your PR
code [1].  You can update that branch using the GitHub UI or locally
from the command line (or with whatever Git client you use).  In this
case, I did:

  $ git checkout condor-fixups
  Switched to branch 'condor-fixups'
  Your branch is up-to-date with 'wking/condor-fixups'.
  $ emacs src/Condor-1.1.xml 
  $ git commit --squash HEAD^ -a
  [condor-fixups 4f4e7b7] squash! Condor-1.1: Use Internet Archive link
   1 file changed, 1 insertion(+)
  $ git rebase -i --autosquash HEAD^^^
  [detached HEAD 21b7b6b] Condor-1.1: Use Internet Archive link
   Date: Thu Aug 3 10:54:41 2017 -0700
   1 file changed, 1 insertion(+)
  Successfully rebased and updated refs/heads/condor-fixups.
  $ git push -f wking condor-fixups
  Counting objects: 8, done.
  Delta compression using up to 8 threads.
  Compressing objects: 100% (8/8), done.
  Writing objects: 100% (8/8), 1.62 KiB | 0 bytes/s, done.
  Total 8 (delta 6), reused 0 (delta 0)
  remote: Resolving deltas: 100% (6/6), completed with 3 local objects.
  To git@github.com:wking/license-list-XML.git
   + d624ca9...c1b6525 condor-fixups -> condor-fixups (forced update)

[1]: https://help.github.com/articles/committing-changes-to-a-pull-request-branch-created-from-a-fork/
